### ISSUE TITLE
TileEntityInput.triggerRedstone() causing NPE with random things

### ISSUE THREAD

**Issue by Builderb0y** [gigabit101/StevesFactoryManager#12]
At [line 54](https://github.com/gigabit101/StevesFactoryManager/blob/1.10.2/src/main/java/vswe/stevesfactory/tiles/TileEntityInput.java#L54), it specifically calls worldObj.getRedstonePower with a null direction. I'm not sure how vanilla handles this, but random things has an ASM patch to the redstone system, and it NPE's with a null direction.

```
Caused by: java.lang.NullPointerException
	at lumien.randomthings.tileentity.redstoneinterface.TileEntityRedstoneInterface.getRedstonePower(TileEntityRedstoneInterface.java:50) ~[TileEntityRedstoneInterface.class:?]
	at lumien.randomthings.handler.AsmHandler.getRedstonePower(AsmHandler.java:303) ~[AsmHandler.class:?]
	at net.minecraft.world.World.func_175651_c(World.java) ~[aid.class:?]
	at vswe.stevesfactory.tiles.TileEntityInput.triggerRedstone(TileEntityInput.java:54) ~[TileEntityInput.class:?]
	at vswe.stevesfactory.blocks.BlockCableInput.updateRedstone(BlockCableInput.java:100) ~[BlockCableInput.class:?]
	at vswe.stevesfactory.blocks.BlockCableInput.func_176213_c(BlockCableInput.java:53) ~[BlockCableInput.class:?]
```

Relavent random things code [here](https://github.com/lumien231/Random-Things/blob/master/src/main/java/lumien/randomthings/tileentity/redstoneinterface/TileEntityRedstoneInterface.java#L50).

**Comment by gigabit101** [comment]
this seems to be his side he has other issue about his redstone on his tracker as well

**Comment by Builderb0y** [comment]
None of which are the same error. The only one that was even close was [#148](https://github.com/lumien231/Random-Things/issues/148), as it was the only one that involved the same ASM hook. Even then though, it NPE'd in an entirely different section of code. The bottom line is that null should never be used as a direction, and the line in question in tileEntityInput is a debug printout anyway. It's a one-line fix with no down sides to removing it.

**Comment by Hirsu** [comment]
I'm having the same issue. Originally, the problem occured when I put a block detector from refined storage on an interface (also from RS), which is why I bugged the RS dev about it. But apparently, the crash is caused by the redstone receiver block right next to it. The weird thing is there are no Random Things items or blocks in my world, as far as I know of at least.

Anyway, if it might help you, the link to my bug report on the RS github is [right here](https://github.com/raoulvdberge/refinedstorage/issues/671). It's got the log, a video showing how I reproduced the error and some more information.
